src/places.scss

```diff
@@ -11,104 +11,104 @@
 // see http://pxtoem.com/ to change/add values
 .algolia-places {
   width: 100%;
+}
```

Comment:
removed .algolia-places nesting, not needed since we already have prefixes everywhere. Also easier to override.